<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.assignment1.testCases.CreaterCustomerTest" tests="6" time="3.943" ignored="0" failures="0" timestamp="06 May 2018 13:28:22 GMT" skipped="3" hostname="sairam-PC" errors="1">
  <testcase name="cancelEdit" time="0.000" classname="com.assignment1.testCases.CreaterCustomerTest">
    <skipped/>
  </testcase> <!-- cancelEdit -->
  <testcase name="ValidateNewUserTitle" time="2.207" classname="com.assignment1.testCases.CreaterCustomerTest"/>
  <testcase name="editCustomerDetails" time="0.000" classname="com.assignment1.testCases.CreaterCustomerTest">
    <skipped/>
  </testcase> <!-- editCustomerDetails -->
  <testcase name="cancelUserCreation" time="1.734" classname="com.assignment1.testCases.CreaterCustomerTest"/>
  <testcase name="readCustomerDetails" time="0.000" classname="com.assignment1.testCases.CreaterCustomerTest">
    <skipped/>
  </testcase> <!-- readCustomerDetails -->
  <testcase name="createUser" time="0.002" classname="com.assignment1.testCases.CreaterCustomerTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.assignment1.testCases.CreaterCustomerTest.createUser(CreaterCustomerTest.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- createUser -->
</testsuite> <!-- com.assignment1.testCases.CreaterCustomerTest -->
